froide/comments/templates/froide_comments/comment.html

```diff
@@ -20,7 +20,7 @@
         {# requester-/moderator-badges #}
         {% if comment.is_requester %}
           <span class="badge text-bg-blue-20 ms-1">{% trans "Requester" %}</span>
-        {% elif comment.is_moderator %}
+        {% elif comment.is_moderator and not comment.comment_as_normal_user %}
```

Comment:
Why not just `comment.is_moderation` (or whatever the Boolean field on the comment should be named)? The permissions of the user are no longer relevant when the comment is a moderation comment, right?

Reply:
If we turn the flag around, yes